katas/metal-slug/alejandra/Vehicle.js

```diff
@@ -0,0 +1,45 @@
+class Vehicle {
+    constructor() {}
+
+    setHealth(health) {
+        this.health = health;
+    }
+
+    getHealth() {
+        return this.health;
+    }
+
+    reciveDamage(damage) {
+        const newHealth = this.health - damage;
+        newHealth >= 0 ? (this.health = newHealth) : (this.health = 0);
+    }
+}
+
+class Tank extends Vehicle {
+    constructor() {
+        super();
+        this.setHealth(3);
+    }
+}
+
+class Plain extends Vehicle {
```

Comment:
Just figured out, I had a typo in the README file, this one should be `Plane` instead of `Plain` 🤣 my bad! could you please add that fix here and in the README file?